httpapi/userapi/userapi.go

```diff
@@ -0,0 +1,31 @@
+package userapi
+
+import (
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/extensions/transferdetails"
+	"github.com/centrifuge/go-centrifuge/nft"
+	"github.com/go-chi/chi"
+)
+
+const (
+	documentIDParam = "document_id"
+	transferIDParam = "transfer_id"
+)
+
+// Register registers the core apis to the router.
+func Register(r chi.Router,
+	docSrv documents.Service,
+	nftSrv nft.Service,
+	transferSrv transferdetails.Service) {
+	h := handler{
+		tokenRegistry: nftSrv.(documents.TokenRegistry),
+		srv:           Service{docSrv: docSrv, transferDetailsService: transferSrv},
+	}
+	r.Post("/documents/{"+documentIDParam+"}/extensions/transfer_details", h.CreateTransferDetail)
```

Comment:
any specific reason for adding `extensions` to the url path ?
IMO, we dont need this to make it consistent with the funding API

Reply:
if all userapis will be extensions, then you are right and this is redundant